cloud_run_service_pubsub/main.tf

```diff
@@ -33,10 +44,6 @@ resource "google_cloud_run_service_iam_binding" "binding" {
 # [END cloudrun_service_pubsub_run_invoke_permissions]
```

Comment:
The Google Cloud Run Service Agent service account (service-<project_number>@serverless-robot-prod.iam.gserviceaccount.com) should have the iam.serviceAccounts.getAccessToken permission via the Cloud Run Service Agent role on the project.

I fixed this by running the following gcloud command, but we should add the TF equivalent to this sample.

gcloud projects add-iam-policy-binding <project_name> --member=serviceAccount:service-<project_number>@serverless-robot-prod.iam.gserviceaccount.com --role=roles/run.serviceAgent